tensorrt_llm/_torch/pyexecutor/finish_reason.py

```diff
@@ -0,0 +1,87 @@
+from tensorrt_llm.bindings.executor import FinishReason
```

Comment:
I guess I would have written something more "pythonic" like a simple enum that maps the FinishStates to the respective FinishReasons, and then have the enum have a "is_finished" method which just returns true if self is one of the three finish values, and just skipped the init setters and getters entirely. But thats a code style thing (also, yours has an added value of a more verbatim port to the cpp class), not fundamental, so I don't think I'm neccesarily correct. At your discretion